assignments/closure.js

```diff
@@ -1,20 +1,46 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
```

Comment:
closure is when you access a variable outside your function that is inside a another function. heres an example.
```suggestion
function sayHello() {
  let name = "Jim";
  function slang() {
    // accessing myName outside this function using a closure
    console.log("Yo " + name)
  }
  // calling slang inside sayHello()
  slang();
}
console.log(sayHello());
```